### DISCUSSION TITLE
Emacs - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#438] [Announcements]
# Emacs - CS自学指南

CS自学指南

https://csdiy.wiki/%E5%BF%85%E5%AD%A6%E5%B7%A5%E5%85%B7/Emacs/

**Comment by lllllliuji** [comment]
我想知道，大佬这些课全部刷过吗，我只刷了一小部分，刷完可太强了

**Reply by zhuansgengbchaoyeu985** [reply]
对刷完可太强了

**Reply by SuperGODOG** [reply]
推荐下载个机场,我在用15一个月,几乎不会机器人验证.当然诚实一点我也有佣金.https://in.mesl.cloud/#/register?code=KxtStC3L

**Reply by CharlesWolff6** [reply]
点我头像，看不同机场套餐对比和推荐^^

**Reply by Cpt11** [reply]
[推荐一个机场，可以试用，不贵](https://bafang.vip/#/register?code=0mobpNZJ)

**Reply by Flora-2017** [reply]
他也许没都看，但肯定都了解过，就像你也许没杀过猪，但你肯定吃过猪肉

**Comment by KingofKanya** [comment]
看到的最好的一大大大篇文章了，很有帮助，万分感谢

**Comment by Kiid-A** [comment]
想问一下有关于STEM32单片机的教程吗

**Reply by xionghuaiyvzhou** [reply]
同问

**Reply by calyle** [reply]
这里是cs，电子的去别处:smile:

**Reply by t1nD1** [reply]
别处去哪

**Comment by SuperGODOG** [comment]
推荐下载个机场,我在用15一个月,几乎不会机器人验证.当然诚实一点我也有佣金.https://in.mesl.cloud/#/register?code=KxtStC3L

**Reply by lismos114** [reply]
谁问你了

**Reply by CS-Trekker** [reply]
DOG，get out!

**Comment by HanChun** [comment]
作者的 文字 逻辑，我读着好舒服；确实有时候 我发现 根本 不是我学不会， 而是 讲的 人没有讲清楚。

**Comment by Zk202011** [comment]
您好 麻烦问一下那个CS 61A的recording部分需要伯克利大学的学生账号登陆才可以看是么

**Reply by leigozi** [reply]
搞了半天"只有被邀请的客人才可以访问此应用程序，如果你想被邀请，请联系你的学校赞助商"麻了

**Reply by wang-zixuan** [reply]
不用，找公开的就行。https://inst.eecs.berkeley.edu/~cs61a/fa18/ 例如这个18fall的video链接都是油管的

**Reply by SuperGODOG** [reply]
需要邀请才能进入ucb,全球都是这个模式

**Reply by buhuizuocaigou** [reply]
正在刷课ing 学到一堆知识

**Reply by fwangapi** [reply]
B站就有61A的视频，开始学习了。还同时看61C也相当不错，用C讲体系结构入门知识。如果有同自学者，欢迎一起讨论作业

**Comment by fwangapi** [comment]
1 秒钟前

B站就有61A的视频，开始学习了。还同时看61C也相当不错，用C讲体系结构入门知识。如果有同自学者，欢迎一起讨论作业

**Reply by leigozi** [reply]
机翻是真的痛苦,作业怎么找

**Comment by XUJINGNING** [comment]
cppinsights 语法糖解糖
compile explorer 查案例的汇编

**Comment by xunxunzi** [comment]
分享低价好用的梯子，接力领佣金https://uuclm.net/#/register?code=ZSPKT2MA

**Comment by a523** [comment]
作者是北大学子? 所以我是在跟北大学子一起学习？并且北大学哥还手把手教我怎么学？是不是全学完？我也有北大学子那么优秀？ 😺。。。

**Reply by ormasia** [reply]
必须😁😁😁😁😁😁😁😁

**Comment by bethewind** [comment]
手机上giscus评论好像看不了,需要翻墙？

**Comment by lotefish** [comment]
求eediy呜呜呜

**Reply by Chen-Zhefeng** [reply]
+1😭

**Reply by CS-Trekker** [reply]
在这求有啥用😂

**Comment by CainLin219** [comment]
最近在学习C语言的debug，看到好多教程里面有提到汇编语言对调试的帮助，请问有什么推荐的国外的汇编语言学习课程吗，我没有在这里找到类似的课程~~~~

**Reply by xielinhegel** [reply]
王爽的《汇编语言》据说很不错

**Reply by curbe454** [reply]
可以看看知乎韦易笑的回答：[终端调试哪家强？](https://zhuanlan.zhihu.com/p/32843449)
<https://zhuanlan.zhihu.com/p/32843449>

**Comment by benyaxianyou** [comment]
想问一下有数字系统基础的课程吗

**Comment by Ziqi-Yang** [comment]
个人觉得emacs相对于vim最大的不足是性能的问题。有些时候会有卡顿，单线程和gc（主要问题）方面的不足。emacs除了有evil（模拟vim插件），也有其他多模态的编辑模式比如meow（个人在用，任何motion都是选择状态，和helix类似）。vim用户现在流行的emacs入门方向是用 doom emacs。

**Comment by TephrocactusMYC** [comment]
现在流行把这些 editor 配成 IDE，B站有个老哥讲的还行，如果比较闲又愿意深入 EMACS 的话可以去看一看。

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287

**Comment by lindsayo11** [comment]
推荐个梯子，自用一年100，每月100G，感觉良好，https://nicehcloud.com/register?code=Md5UAxPk

**Comment by moonloopy** [comment]
我这里有推荐的梯 好用稳定不掉线 需要的话可以找我 邮箱:moonloopy@linux.do

**Comment by n04h1n9** [comment]
Ctrl键可以不用改，用手掌来摁，习惯之后非常舒服，小拇指压力为0，我力气小，摁esc摁多了手指还是会痛，用手掌摁则完全没感觉
